boards/nxp/mr-canhubk3/nuttx-config/include/board.h

```diff
@@ -152,6 +152,18 @@
 #define PIN_LPUART1_RX    (PIN_LPUART1_RX_3 | PIN_INPUT_PULLUP)   /* PTC6 */
 #define PIN_LPUART1_TX    PIN_LPUART1_TX_3   /* PTC7 */
 
+
+/* LPUART4 /dev/ttyS3  P8B 3X7 Pin 3 Single wire RC UART */
+
+#define PIN_LPUART4_RX    PIN_LPUART4_TX_3   /* PTE11 */
```

Comment:
Let's Add a comment describing why this done this way and is not a typo

Reply:
Done